R/connection_spark.R

```diff
@@ -192,6 +195,24 @@ spark_connect <- function(master,
   } else if (method == "databricks") {
     scon <- databricks_connection(config = config,
                                   extensions)
+  } else if (method == "qubole") {
+    scon <- shell_connection(master = master,
```

Comment:
Instead of creating the `shell_connection` here, create a helper function `qubole_connection()` in a new `qubole_connection.R` file and move this code into it.

Reply:
Actually... if you don't need changes at all in how the connection is initialized compared to the shell connection... you can also consider removing this code and changing instead:

```
if (method == "shell") {
```

into

```
if (method == "shell" || method == "qubole") {
```